data-prepper-pipeline-parser/src/resources/templates/documentdb-template.yaml

```diff
@@ -38,16 +38,16 @@
         sts_role_arn: "<<$.<<pipeline-name>>.source.documentdb.opensearch.aws.sts_role_arn>>"
         sts_external_id: "<<$.<<pipeline-name>>.source.documentdb.opensearch.aws.sts_external_id>>"
         sts_header_overrides: "<<$.<<pipeline-name>>.source.documentdb.opensearch.aws.sts_header_overrides>>"
-        bucket: "<<$.<<pipeline-name>>.source.documentdb.s3_bucket>>"
-        threshold:
+      bucket: "<<$.<<pipeline-name>>.source.documentdb.s3_bucket>>"
+      threshold:
           event_collect_timeout: "30s"
           maximum_size: "1mb"
-        aggregate_threshold:
+      aggregate_threshold:
           maximum_size: "128mb"
           flush_capacity_ratio: 0
-        object_key:
-          path_prefix: "${getMetadata(\"s3_partition_key\")}"
-        codec:
+      object_key:
+          path_prefix: "local-test/${getMetadata(\"s3_partition_key\")}"
+      codec:
           json:
 
   "<<pipeline-name>>.s3":
```

Comment:
Shouldn't have indent here because this is top level pipeline?

Reply:
Thanks for noticing the indentation, ll fix it.